config/formats.ts

```diff
@@ -626,8 +626,7 @@ export const Formats: FormatList = [
 			`&bullet; <a href="https://www.smogon.com/forums/threads/3679692/">Pok&eacute;bilities</a>`,
 		],
 
-		mod: 'pokebilities',
-		ruleset: ['Standard', 'Dynamax Clause'],
+		ruleset: ['Standard', 'Dynamax Clause', 'Pokebilities Rule'],
```

Comment:
You don't need to add `Rule` after something like that, btw. I don't think it adds anything?

We already have disambiguation where `pokebilities` is the rule and `gen8pokebilities` is the format.